testing/libfort/APKBUILD

```diff
@@ -1,28 +1,22 @@
 # Maintainer: Pete Dietl <petedietl@gmail.com>
 pkgname=libfort
-pkgver=0.1.3
+pkgver=0.1.4
 pkgrel=0
 pkgdesc="A simple crossplatform library to create formatted text tables."
 url="https://github.com/seleznevae/libfort"
 arch="all"
 license="MIT"
-makedepends="cmake binutils-gold"
+makedepends="cmake"
 subpackages="$pkgname-dev"
 source="$pkgname-$pkgver.tar.gz::https://codeload.github.com/seleznevae/libfort/tar.gz/v$pkgver"
-builddir="$srcdir/$pkgname-$pkgver"
-
-prepare() {
-	default_prepare
-	mkdir -p "$builddir/build"
-}
 
 build() {
-	cd "$builddir/build"
+	mkdir build && cd build
 	if [ "$CBUILD" != "$CHOST" ]; then
 		CMAKE_CROSSOPTS="-DCMAKE_SYSTEM_NAME=Linux -DCMAKE_HOST_SYSTEM_NAME=Linux"
 	fi
 
-	cmake .. \
+	cmake "$builddir" \
```

Comment:
`cmake "$builddir" \`

is the same as
`cmake .. \`

Reply:
TRU